---
layout: post
title: Rapid evolution of reproduction-related transcripts
subtitle: 1st Thesis chapter published!
bigimg: /img/BMC_Fig_1.png
---

The first chapter of my PhD Thesis has been published! For this project I inherited a dataset that was produced before I joined the lab. 
The data consisted of large RNA-Seq dataset for three *Macrostomum* species (*M. hystrix*, *M. spirale* and *M. pusillum*) with high sequencing depth across multiple biological conditions. We had RNA-Seq for adults, juveniles and for regenerant. Regenerants are a mixture of animals that were collected at increasing time post amputation of the tail. It was clear from the beginning that we wanted to use annotations from *M. lignano* to annotate groups of orthologous genes and then look at rate of evolution and gene expression.
However, since the data was originally collected with the main aim to produce high-quality transcriptomes some of the biological condidtions were not comparable between species. Specifically, we found that the *M. pusillum* juveniles were already quite advanced in developement making the Hatchling - Adult tricky to compare across the genus. You can see this in the picture below where gene expression between hatchlings and adults compared. In both *M. hystrix* and *M. spirale* there is a large cloud to the right of the diagonal line representing transcripts with higher expression in adults, but in *M. pusillum* this cloud is shifted closer to the diagonal. 
![DE Hatchling vs Adults](/img/BMC_Fig_3_crop.png)

Because of this we had to abandon our analysis of gene expression evolution and focused on sequence based measures and homology detection failure. We transferred annotations from *M. lignano* to the other species, which allowed us to identify reproduction-related genes. Due to male-female coevolution these genes are expected to diverge rapidly but this has not been looked at in flatworms. We could show that reproduction-related genes have a higher sequence divergence and are also harder to detect with phylogenetic distance. 
Therefore, we can say that sexual selection likely also drives rapid evolution of these genes in flatworms. I'm very happy we could find such an exciting result given the initially trouble with the data. You can read the paper [here](https://rdcu.be/b5tcA) if you want to learn more about this. 



